### ISSUE TITLE
move codebase to subfolder

### ISSUE THREAD

**Issue by jokiefer** [mrmap-community/mrmap#44]
### Proposed Changes
move MrMap codebase to a subfolder called `src` or `main` 

### Justification

1. Sonarcloud has trouble by separating test from source files:
```
ERROR: Error during SonarScanner execution
ERROR: File tests/test_data.py can't be indexed twice. Please check that inclusion/exclusion patterns produce disjoint sets for main and test files
ERROR: 
ERROR: Re-run SonarScanner using the -X switch to enable full debug logging.

```
2. we get a compact project page, where the readme shows quickly up

@jansule Take care of this. I will do it tomorrow. So better wait for it to avoid merge conflicts.

**Comment by jansule** [comment]
+1 on moving the codebase to a subfolder

**Comment by jokiefer** [comment]
merged in #46
